test/passing/doc_comments.mli

```diff
@@ -165,3 +165,7 @@ val x : x
         + a
         + b
         + c}} *)
+
+(** {{:https://google.fr/} Google} *)
+
+(** {:https://google.fr/} *)
```

Comment:
Can I ask to change this to github.com just to not match some regexes that will crawl over this code